CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ minor release, the component will be purged, so be prepared (see `Updating` sect
 - New storage component for small objects named Peapod (#2453)
 - New `blobovnicza-to-peapod` tool providing blobovnicza-to-peapod data migration (#2453)
 - SN's version and capacity is announced via the attributes automatically but can be overwritten explicitly (#2455, #602)
+- `peapod` command to work with Peapods from NeoFS Lens (#2507)
```

Comment:
i _can_ what is meant here but wording like "`neofs-lens` has a new `peapod` subcommand" would be more understandable, IMO

Reply:
fixed this and commit msg